internal/dataplane/configfetcher/config_fetcher_test.go

```diff
@@ -0,0 +1,320 @@
+package configfetcher_test
```

Comment:
I think this file should be renamed to `kongrawstate_test.go` after the reorganisation.